Source/HarmonyPatches/Patches_Room/Patch_Room.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using HarmonyLib;
+using PrisonLabor.Core.Trackers;
+using Verse;
+
+namespace PrisonLabor.HarmonyPatches.Patches_Rooms
+{
+    [StaticConstructorOnStartup]
```

Comment:
Why just not normal harmony patching by annotation?

Reply:
It's mirrored in an other project of mine used for profiling (in which i use dub analyzer ```[PerformancePatch]``` ) which require this setup.